webapp/webpack.config.babel.js

```diff
@@ -52,13 +53,28 @@ let config = {
       },
       {
         test: /\.css$/,
-        exclude: /node_modules/,
-        loaders: ['style', 'css?modules&sourceMap&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]', 'postcss']
+        include: path.resolve(__dirname, "src"),  // limit match to only src/
+        exclude: path.resolve(__dirname, "src/public.css"),
+        loader: ExtractTextPlugin.extract(
+          'style',
+          'css?modules&sourceMap&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]',
+          'postcss'
+        )
+      },
+      {
+        test: path.resolve(__dirname, "src/public.css"),
```

Comment:
The public.css should get a hash bc it will change and should therefore have the cache busted (as your PR successfully does).  I've just been using the public.css file for any situation where you want an old-school CSS file to workaround the webpack CSS stuff (which you sometimes need to do, e.g. for an override).

I was thinking the public directory would be any file that is available to the user without modification, and that would include the public.css file.  However, I guess it's _kind of_ being modified (the name but not the contents).  I thought it made sense to put it in public, but I guess it's not super clear and doesn't really matter.